bokehjs/src/coffee/core/util/object.ts

```diff
@@ -27,6 +29,21 @@ export function clone<T>(obj: T): T {
   return extend({}, obj)
 }
 
+export function merge<T>(obj1: {[key: string] : Array<T>}, obj2: {[key: string]: Array<T>}): object {
```

Comment:
> Also, I don't know if there is common notion of what "merge" should mean for objects.

I was actually going to suggest it *not* go in `util/objects` for this reason (tho I think tests for it are good)